src/dashboard/src/installer/steps.py

```diff
@@ -53,19 +53,22 @@ def set_agent_code(agent_code):
 
 
 def setup_pipeline(org_name, org_identifier):
-    # Assign UUID to Dashboard
-    dashboard_uuid = str(uuid.uuid4())
-    helpers.set_setting('dashboard_uuid', dashboard_uuid)
+    dashboard_uuid = helpers.get_setting('dashboard_uuid')
+    # Setup pipeline only if dashboard_uuid doesn't already exists
+    if not dashboard_uuid:
```

Comment:
We try to avoid nesting code when possible, i.e. try to return early.

```python
def setup_pipeline(org_name, org_identifier):
    dashboard_uuid = helpers.get_setting('dashboard_uuid')
    # Setup pipeline only if dashboard_uuid doesn't already exists
    if dashboard_uuid:
        return
    # ... continue here.